### ISSUE TITLE
Unmet dependancy errors

### ISSUE THREAD

**Issue by 34r7h** [lupas/nuxt-fire#65]
Hi I'm facing some challenges using this module. After installing nuxt-fire, I got errors that firebase was missing so I yarn add firebase. Now, I'm getting core-js errors like this..

` ERROR  Failed to compile with 34 errors

These dependencies were not found:

* core-js/modules/es6.array.find in ./functions/.nuxt/client.js
* core-js/modules/es6.array.iterator in ./functions/.nuxt/client.js
* core-js/modules/es6.date.to-string in ./functions/.nuxt/utils.js, ./functions/.nuxt/components/nuxt.js
* core-js/modules/es6.function.name in ./functions/.nuxt/index.js
* core-js/modules/es6.object.assign in ./functions/.nuxt/client.js
* core-js/modules/es6.object.keys in ./functions/.nuxt/utils.js
* core-js/modules/es6.object.to-string in ./functions/.nuxt/router.scrollBehavior.js, ./functions/.nuxt/components/nuxt-link.client.js
* core-js/modules/es6.promise in ./functions/.nuxt/client.js
* core-js/modules/es6.regexp.constructor in ./functions/.nuxt/utils.js
* core-js/modules/es6.regexp.match in ./functions/.nuxt/client.js
* core-js/modules/es6.regexp.replace in ./functions/.nuxt/utils.js, ./functions/.nuxt/components/nuxt.js
* core-js/modules/es6.regexp.search in ./functions/.nuxt/utils.js
* core-js/modules/es6.regexp.split in ./functions/.nuxt/utils.js, ./node_modules/babel-loader/lib??ref--2-0!./node_modules/vue-loader/lib??vue-loader-options!./functions/.nuxt/components/nuxt-build-indicator.vue?vue&type=script&lang=js&
* core-js/modules/es6.regexp.to-string in ./functions/.nuxt/utils.js, ./functions/.nuxt/components/nuxt.js
* core-js/modules/es6.string.includes in ./functions/.nuxt/client.js, ./functions/.nuxt/components/nuxt-link.client.js
* core-js/modules/es6.string.iterator in ./functions/.nuxt/utils.js
* core-js/modules/es6.string.repeat in ./functions/.nuxt/utils.js
* core-js/modules/es6.string.starts-with in ./functions/.nuxt/utils.js
* core-js/modules/es6.symbol in ./functions/.nuxt/utils.js, ./functions/.nuxt/components/nuxt-link.client.js
* core-js/modules/es7.array.includes in ./functions/.nuxt/client.js, ./functions/.nuxt/components/nuxt-link.client.js
* core-js/modules/es7.object.get-own-property-descriptors in ./functions/.nuxt/index.js
* core-js/modules/es7.promise.finally in ./functions/.nuxt/client.js
* core-js/modules/es7.symbol.async-iterator in ./functions/.nuxt/client.js, ./functions/.nuxt/components/nuxt-link.client.js
* core-js/modules/web.dom.iterable in ./functions/.nuxt/axios.js, ./functions/.nuxt/components/nuxt-link.client.js

To install them, you can run: npm install --save core-js/modules/es6.array.find core-js/modules/es6.array.iterator core-js/modules/es6.date.to-string core-js/modules/es6.function.name core-js/modules/es6.object.assign core-js/modules/es6.object.keys core-js/modules/es6.object.to-string core-js/modules/es6.promise core-js/modules/es6.regexp.constructor core-js/modules/es6.regexp.match core-js/modules/es6.regexp.replace core-js/modules/es6.regexp.search core-js/modules/es6.regexp.split core-js/modules/es6.regexp.to-string core-js/modules/es6.string.includes core-js/modules/es6.string.iterator core-js/modules/es6.string.repeat core-js/modules/es6.string.starts-with core-js/modules/es6.symbol core-js/modules/es7.array.includes core-js/modules/es7.object.get-own-property-descriptors core-js/modules/es7.promise.finally core-js/modules/es7.symbol.async-iterator core-js/modules/web.dom.iterable`

**Comment by aseidma** [comment]
Hey @34r7h ,

I encountered this error myself, even without using nuxt-fire. As far as I know it is an issue related to the naming of folders in core-js@3<, as they are then only named modules/es.array.iterator for example, instead of containing info about the es version (as seen in the error output).

What helped fix it for me is uninstalling core-js and reinstalling it like so:

`yarn add core-js@2`
or for npm
`npm install core-js@2`

This installs an older version of core-js and therefore fixes the naming of the folders. Although this is more of a quickfix than a final solution, I have encountered no problems using this setup.
